### PR TITLE
adds gladius

### PR DESCRIPTION
**Pull request by rudolfix**

### CODE REVIEW

File: src/icos_config/gladius.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { toPromise } from '../utils';
+import { convertWeb3Value } from '../utils/web3';
+
+export default {
+  // crowdSaleTokenContract: '0x57BFfFD48366F78e787e167419C8c05CDb849EdE',
+  // tokenContract: '0x4632d1c31c5D9E28E84eAE0173B3aFc9acA81aC8',
```
Comment (MostafaBalata) [inline src/icos_config/gladius.js]: Why are you leaving these commented code?

File: src/icos_config/gladius.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    logo: 'https://gladius.io/favicon/android-icon-192x192.png',
+  },
+  events: {
+    Transfer: {
+      args: {
+        tokens: '_value',
+        sender: '_to',
+      },
+      customArgs: {
+        // _from: '0x4632d1c31c5D9E28E84eAE0173B3aFc9acA81aC8',
```
Comment (MostafaBalata) [inline src/icos_config/gladius.js]: Why are you leaving these commented code?

File: src/icos_config/gladius.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        _from: '0x71d01db8d6a2fbea7f8d434599c237980c234e4c'
+      },
+      firstTransactionBlockNumber: 4365348,
+      lastTransactionBlockNumber: null,
+      maxBlocksInChunk: 52960,
+      countTransactions: true,
+      // address: '0x4632d1c31c5D9E28E84eAE0173B3aFc9acA81aC8'
```
Comment (MostafaBalata) [inline src/icos_config/gladius.js]: commented code again?

File: src/icos_config/gladius.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+      const now = Math.floor(new Date().getTime() / 1000);
+      const begTs = await toPromise(icoContract.start)();
+      const presaleEndTs = await toPromise(icoContract.presaleEnd)();
+      if (now < begTs) {
+        return 'not started';
+      } else if (now >= begTs && now <= presaleEndTs) {
+        return 'in pre-sale';
+      }
+      const stage = await toPromise(icoContract.stage)();
+      if (stage == 3) {
```
Comment (MostafaBalata) [inline src/icos_config/gladius.js]: can you change `==` to `===`?

Comment (rudolfix) [reply src/icos_config/gladius.js]: sure! my bad!

File: src/icos_config/gladius.js (hunk 1 above)
Comment (rudolfix) [reply src/icos_config/gladius.js]: Ok I will clarify why this is left

**Pull request merged**